test/clientmetricreport/DefaultClientMetricReport.test.ts

```diff
@@ -441,17 +454,25 @@ describe('DefaultClientMetricReport', () => {
       const videoStreamMetrics = clientMetricReport.getObservableVideoMetrics();
       expect(Object.keys(videoStreamMetrics).length).to.equal(0);
     });
+
+    it('returns raw RTCStatsReport report', () => {
+      clientMetricReport.rtcStatsReport = {} as RTCStatsReport;
+      const rawMetricReport = clientMetricReport.getRTCStatsReport();
+      expect(rawMetricReport).to.equal(clientMetricReport.rtcStatsReport);
+    });
   });
 
   describe('clone', () => {
     it('clones', () => {
       clientMetricReport.globalMetricReport = new GlobalMetricReport();
       clientMetricReport.streamMetricReports = { 1: new StreamMetricReport() };
+      clientMetricReport.rtcStatsReport = {} as RTCStatsReport;
```

Comment:
nit: Is this needed? isn't the constructor already populated it above?

Reply:
It's not necessary but it serves as a cleanup before starting this unit test, so I think it's good to have it.

Reply:
Ideally you should try to minimize the need to modify the value directly :). If you need to do so then there is probably something wrong with the class :)